pkg/tasks/alertmanager.go

```diff
@@ -178,6 +178,13 @@ func (t *AlertmanagerTask) create(ctx context.Context) error {
 			return errors.Wrap(err, "initializing Alertmanager object failed")
 		}
 
+		// Use console URL as Alertmanager ExternalURL since UI access to AlertManager is disabled.
+		consoleRoute, err := t.client.GetRouteWithNameAndNameSpace(ctx, "console", "openshift-monitoring")
```

Comment:
Should you not be using the openshift-console namespace? openshift-monitoring does not have a console route.

Reply:
ooops, I will correct